docs/src/main/asciidoc/kafka-reactive-getting-started.adoc

```diff
@@ -0,0 +1,502 @@
+Inside the `processor` project locate the `src/main/java/org/acme/kafka/processor/QuoteProcessor.java` file and add the following:
+
+[source, java]
+----
+package org.acme.kafka.processor;
+
+import java.util.Random;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.acme.kafka.model.Quote;
+import org.eclipse.microprofile.reactive.messaging.Incoming;
+import org.eclipse.microprofile.reactive.messaging.Outgoing;
+
+import io.smallrye.reactive.messaging.annotations.Blocking;
+
+/**
+ * A bean consuming data from the "request" Kafka topic and giving out a random quote.
+ * The result is pushed to the "quotes" Kafka topic.
+ */
+@ApplicationScoped
+public class QuotesProcessor {
+
+    private Random random = new Random();
+
+    @Incoming("requests")       // <1>
+    @Outgoing("quotes")         // <2>
+    @Blocking                   // <3>
+    public Quote process(String quoteRequest) throws InterruptedException {
+        // simulate some hard working task
+        Thread.sleep(200);
+        return new Quote(quoteRequest, random.nextInt(100));
+    }
+}
+
+----
+<1> Indicates that the method consumes the items from the `requests` channel
+<2> Indicates that the objects returned by the method are sent to the `quotes` channel
+<3> Indicates that the processing is _blocking_ and cannot be run on the caller thread.
+
+The `process` method is called for every Kafka _record_ from the `quote-requests` topic, and will send a `Quote` object to the `quotes` topic.
+
+As with the previous example we need to configure the connectors in the `application.properties` file:
+
+[source, properties]
+----
+%dev.quarkus.http.port=8081
+
+# Configure the incoming Kafka topic `quote-requests`
+mp.messaging.incoming.requests.connector=smallrye-kafka
+mp.messaging.incoming.requests.topic=quote-requests
+mp.messaging.incoming.requests.auto.offset.reset=earliest
+
+# Configure the outgoing Kafka topic `quotes`
+mp.messaging.outgoing.quotes.connector=smallrye-kafka
+mp.messaging.outgoing.quotes.value.serializer=io.quarkus.kafka.client.serialization.ObjectMapperSerializer
+----
+
+Note that in this case we have one incoming and one outgoing connector configuration, each one distinctly named.
+The configuration keys are structured as follows:
+
+`mp.messaging.[outgoing|incoming].{channel-name}.property=value`
+
+The `channel-name` segment must match the value set in the `@Incoming` and `@Outgoing` annotation:
+
+* `quote-requests` -> Kafka topic from which we read the quote requests
+* `quotes` -> Kafka topic in which we write the quotes
+
+[NOTE]
+====
+More details about this configuration is available on the https://kafka.apache.org/documentation/#producerconfigs[Producer configuration] and https://kafka.apache.org/documentation/#consumerconfigs[Consumer configuration] section from the Kafka documentation. These properties are configured with the prefix `kafka`. An exhaustive list of configuration properties is available in <<kafka-configuration>>.
```

Comment:
```suggestion
More details about this configuration is available on the https://kafka.apache.org/documentation/#producerconfigs[Producer configuration] and https://kafka.apache.org/documentation/#consumerconfigs[Consumer configuration] section from the Kafka documentation. These properties are configured with the prefix `kafka`. A list of the most used configuration properties is available in <<kafka-configuration>>.
```

Reply:
The kafka configuration section should probably be in the reference guide (and so use xref)